runscript.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker compose up -d influxdb grafana
+docker compose run --rm -i --volume "$(pwd)/out":/out --workdir "/out" k6 run $1
```

Comment:
does docker make a directory when you reference it like this and it doesn't exist? or do we need a `mkdir -p out` in here too?

Reply:
So Docker will create the directory, however when Docker created it, there was some gross permission problems when trying to remove the directory. Adding a `mkdir` command to the script completely fixed that, so I will submit a commit that includes that. Thanks for bringing this up!